src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -239,335 +607,873 @@ private static void init(){}
+        public static final TagKey<Item> FOODS_VEGETABLE = cTag("foods/vegetable");
         /**
-         * Blocks which are often replaced by stone ores, i.e. the ores in the tag {@link #ORES_IN_GROUND_STONE}, during world generation
+         * Strawberries, raspberries, and other berry foods belong in this tag.
+         * Cherries would NOT go here as they are considered a "stone fruit" within culinary fields.
          */
-        public static final TagKey<Item> ORE_BEARING_GROUND_STONE = tag("ore_bearing_ground/stone");
+        public static final TagKey<Item> FOODS_BERRY = cTag("foods/berry");
+        public static final TagKey<Item> FOODS_BREAD = cTag("foods/bread");
+        public static final TagKey<Item> FOODS_COOKIE = cTag("foods/cookie");
+        public static final TagKey<Item> FOODS_RAW_MEAT = cTag("foods/raw_meat");
+        public static final TagKey<Item> FOODS_COOKED_MEAT = cTag("foods/cooked_meat");
+        public static final TagKey<Item> FOODS_RAW_FISH = cTag("foods/raw_fish");
+        public static final TagKey<Item> FOODS_COOKED_FISH = cTag("foods/cooked_fish");
         /**
-         * Ores which on average result in more than one resource worth of materials
+         * Soups, stews, and other liquid food in bowls belongs in this tag.
          */
-        public static final TagKey<Item> ORE_RATES_DENSE = tag("ore_rates/dense");
+        public static final TagKey<Item> FOODS_SOUP = cTag("foods/soup");
         /**
-         * Ores which on average result in one resource worth of materials
+         * Sweets and candies like lollipops or chocolate belong in this tag.
          */
-        public static final TagKey<Item> ORE_RATES_SINGULAR = tag("ore_rates/singular");
+        public static final TagKey<Item> FOODS_CANDY = cTag("foods/candy");
         /**
-         * Ores which on average result in less than one resource worth of materials
+         * Pies and other pie-like foods belong in this tag.
          */
-        public static final TagKey<Item> ORE_RATES_SPARSE = tag("ore_rates/sparse");
-        public static final TagKey<Item> ORES = tag("ores");
-        public static final TagKey<Item> ORES_COAL = tag("ores/coal");
-        public static final TagKey<Item> ORES_COPPER = tag("ores/copper");
-        public static final TagKey<Item> ORES_DIAMOND = tag("ores/diamond");
-        public static final TagKey<Item> ORES_EMERALD = tag("ores/emerald");
-        public static final TagKey<Item> ORES_GOLD = tag("ores/gold");
-        public static final TagKey<Item> ORES_IRON = tag("ores/iron");
-        public static final TagKey<Item> ORES_LAPIS = tag("ores/lapis");
-        public static final TagKey<Item> ORES_NETHERITE_SCRAP = tag("ores/netherite_scrap");
-        public static final TagKey<Item> ORES_QUARTZ = tag("ores/quartz");
-        public static final TagKey<Item> ORES_REDSTONE = tag("ores/redstone");
+        public static final TagKey<Item> FOODS_PIE = cTag("foods/pie");
```

Comment:
This tag exists in NeoForge but not in Fabric. Please remove it.

Reply:
It's going to exist in Fabric the PR is waiting to be merged
https://github.com/FabricMC/fabric/pull/4114